### ISSUE TITLE
Youtube left sidebar doesn't scroll

### ISSUE THREAD

**Issue by 11plustwo** [yourduskquibbles/webannoyances#482]
### URL(s) where the issue occurs

`https://www.youtube.com/`

### Describe the issue

In Youtube, there is a left sidebar which shows, among other things, the channels you are subscribed to. With WebAnnoyances enabled, the left sidebar does not scroll independently like it normally does. Instead it scrolls along with the rest of the page. However if your cursor is over the left sidebar, no scrolling happens. Your cursor must be over the rest of the page to scroll. Normally, if your cursor is over the left sidebar and you scroll, the left sidebar will scroll without the rest of the page. Also, if you subscribe to more than 7 channels, only 7 will be displayed and there will be a 'show X more' link. If you click 'show X more', the 'show X more' link does not disappear like it normally does.

### Screenshot(s)

I couldn't think of a good way to screen shot the lack of scrolling.

<img width="871" alt="Screen Shot 2022-09-15 at 4 44 55 PM" src="https://user-images.githubusercontent.com/4984743/190706019-a98c36d0-9a2c-4d85-b631-4c1454380c3f.png">

<img width="776" alt="Screen Shot 2022-09-15 at 4 45 26 PM" src="https://user-images.githubusercontent.com/4984743/190706136-ce2a08a3-ff14-4499-9a75-f87ff54d96c4.png">

### Versions

- Browser/version: Chrome/Version 105.0.5195.102 (Official Build) (x86_64)
- uBlock Origin version: uBlock Origin 1.44.2

### Settings

I have a filter for trafficjunky.com. Otherwise have made no changes.

### Notes

I'm not really sure what to put here. I'm a new user of WebAnnoynces so forgive me if I'm missing something obvious. I did try both regular and incognito mode in Chrome and had the same result.

Thank you and let me know if there's anything else I can provide.

**Comment by BlazDT** [comment]
@yourduskquibbles any news?

**Comment by 11plustwo** [comment]
I never heard back. Has a change been made? I never tried again as I never got a response.

**Comment by Eta-Beta** [comment]
Hello, I too would be interested in solving this problem. Thank you.

**Comment by Sonic5435325** [comment]
This is how the website behaves even without the list.
This version isn't being maintained anymore, please report any problems you find to the new fork, https://github.com/LanikSJ/webannoyances.

**Comment by 11plustwo** [comment]
No, this is not how the website behaves without WebAnnoyances. But I will open a ticket on the new fork.

**Comment by Sonic5435325** [comment]
My bad I had to reload the page with cosmetic filtering off to see it, I'll try to find a solution

**Comment by 11plustwo** [comment]
The issue does not appear to be a problem with the fork you provided. Thanks

**Comment by 11plustwo** [comment]
I spoke too soon. I'm experencing similar but different issues with the fork. I'm just not going to bother further

**Comment by Sonic5435325** [comment]
> I spoke too soon. I'm experencing similar but different issues with the fork. I'm just not going to bother further

Please make an issue there, then. Issues can't really be fixed unless they're reported (Or found separately)
